package.json

```diff
@@ -17,20 +17,20 @@
     "prettify:diff": "yarn run prettier --config .prettierrc.js --list-different '**/*.+(ts|tsx|js|jsx|sol|java)'"
   },
   "dependencies": {
-    "@celo/connect": "1.1.0",
-    "@celo/contractkit": "1.1.0",
-    "@celo/utils": "1.1.0",
-    "@celo/wallet-hsm-aws": "1.1.0",
-    "@celo/wallet-hsm-azure": "1.1.0",
+    "@celo/connect": "1.5.2",
+    "@celo/contractkit": "1.5.2",
+    "@celo/utils": "1.5.2",
+    "@celo/wallet-hsm-aws": "1.5.2",
+    "@celo/wallet-hsm-azure": "1.5.2",
     "bignumber.js": "^8.1.1",
-    "bunyan": "^1.8.12",
```

Comment:
Why are we removing some carets? (I'm not against it, I just want to understand the rationale.)

Reply:
It was actually autogenerated when I did `yarn upgrade`. I think better to leave this way so that we can check for vulnerabilities before going for an specific version?

Reply:
Agreed. Thanks!